phpBB/adm/style/admin.js

```diff
@@ -246,6 +246,57 @@ function parse_document(container)
 	});
 }
 
+async function send_ajax_request(elem)
+{
+	const method = (elem.hasAttribute('data-method')) ? elem.getAttribute('data-method') : 'get';
+	const params = new URLSearchParams();
+	if (method === 'post')
+	{
+		for (const attr of elem.attributes) 
+		{
+			if (attr.name.startsWith('data-post-'))
+			{
+				params.append(attr.name.substr(10).replace('-', '_'), attr.value);
+			}
+		}
+	}
+	
+	const response = await fetch(elem.getAttribute('data-url'), {
+		method: method,
+		body: params,
+	});
+	if (!response.ok)
+	{
+		phpbb.alert(response.status, response.statusText);
```

Comment:
Was the idea to just output the status id and the text without any language parsing?

Reply:
yeah, pretty much.